---
zaber_stage/PoseAndDebugInfo pose_and_debug_info
